### PR TITLE
Link updates and add new sample - Performace Chart

### PR DESCRIPTION
**Pull request by Lupusa87**
This sample has private github repo, I added only live demo link.

  https://lupblazordemos.z13.web.core.windows.net/PerformanceChartPage

### CODE REVIEW

**Comment by guardrex** [comment]
Look good ....... there isn't a GitHub repo for the Blazor Performance Chart?

**Comment by Lupusa87** [comment]
There is but private, I want to keep it private if open source is not requirement.

**Comment by guardrex** [comment]
Fair enough. :+1:

File: community.md [hunk 1]
```diff
@@ -64,21 +64,22 @@ Below is a list of projects and resources from the Blazor community (links in ea
...
 * [Blazor Hacker News Clone](https://github.com/lohithgn/blazor-hackernews-clone) ([Demo](http://blazorhackernews.surge.sh/)) &ndash; A Hacker News clone built in Blazor.
 * [Blazor Loan Calculator](https://github.com/Lupusa87/BlazorLoanCalculator) ([Demo](https://lupblazordemos.z13.web.core.windows.net/LoanCalculatorPage)) &ndash; A loan payments calculator developed with Blazor.
 * [Blazor Paint](https://github.com/Lupusa87/BlazorPaint) ([Demo](https://lupblazordemos.z13.web.core.windows.net/PaintPage)) &ndash; Drawing app written in Blazor.
-* [Blazor Password Pattern](https://github.com/Lupusa87/BlazorPasswordPattern) ([Demo](https://lupusassblazordemos.azurewebsites.net)) &ndash; Password pattern based on SVG.
+* [Blazor Password Pattern](https://github.com/Lupusa87/BlazorPasswordPattern) ([Demo](https://lupusablazordemos.azurewebsites.net)) &ndash; Password pattern based on SVG.
```
Comment (danroth27) [inline community.md]: Demo -> https://lupblazordemos.z13.web.core.windows.net/PasswordPatternPage

**Pull request merged**